src/content/docs/tutorial-customer-experience/bottom-funnel-analysis.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: "Analyze the bottom-of-funnel"
+tags:
+  - Observability maturity
+  - Customer experience
+  - Digital customer experience
+  - Implementation guide
+  - Conversions
+  - Funnel
+  - Bottom of the funnel
+metaDescription: "New Relic observability maturity series: Bottom-of-the-funnel analysis shows New Relic users how to improve end-user conversions."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/bofta-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/bofta-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/bottom-funnel-analysis-customer-journey-guide
+---
+
+import cxBotfaFunnel from 'images/cx_diagram_botfa-funnel.webp'
+
+import cxBotfaEcommerce from 'images/cx_diagram_botfa-ecommerce.webp'
+
+import cxBotfaCarInsurance from 'images/cx_diagram_botfa-car-insurance.webp'
+
+import cxBotfaPagesActions from 'images/cx_diagram_pages-actions.webp'
+
+import cxBotfaDashboard from 'images/cx_diagram_botfa-dashboard.webp'
+
+import cxBotfaKpiTracking from 'images/cx_diagram_botfa-kpi-tracking.webp'
+
+Even though [improving customer experience](/docs/tutorial-customer-experience/improve-customer-experience/) is often done comprehensively, you may find that focusing your efforts on improving conversion can also provide a large increase in customer experience quality. This is what's known as bottom-of-the-funnel analysis is about improving conversion by focusing on performance at the end of the user journey instead of the customer experience as a whole. 
```

Comment:
"improving your customer experience"

This paragraph needs to be rewritten. What's the key point here? What will someone get out of reading this doc?

Reply:
Great call out, I've given this section another once-over.